doc/specs/GitHub-discussions-spec.md

```diff
@@ -0,0 +1,49 @@
+# GitHub discussions spec
+
+## What is GitHub Discussions
+
+GitHub Discussions is a feature that currently is in the closed beta status. It will be released until the end of the year. 
```

Comment:
It will be released *at* the end of the year.

Reply:
That's not true. We don't know when it gets released. But at the end of the year it's surely released.

Reply:
BTW i've added two new sections